soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -50,7 +50,11 @@ void z_intel_adsp_ipc_isr(const void *devarg)
 
 		regs->tdr = INTEL_ADSP_IPC_BUSY;
 		if (done && !IS_ENABLED(CONFIG_SOC_INTEL_CAVS_V15)) {
-			regs->tda = INTEL_ADSP_IPC_DONE;
+			#ifdef CONFIG_SOC_SERIES_INTEL_ACE
+				regs->tda = INTEL_ADSP_IPC_ACE1X_TDA_DONE;
+			#else
+				regs->tda = INTEL_ADSP_IPC_DONE;
```

Comment:
cAVS sets TDA = 0x80000000, new behavior!

Reply:
How is that a new behavior?
It is exactly the same behavior as previously:
![image](https://user-images.githubusercontent.com/12482515/190480461-3f0728db-30f0-4d51-91d6-a9ba08f3486f.png)
just moved under #else directive (meaning "everything except ACE").

Reply:
Walk me through this?  That's not how I'm reading it.  In the code in the tree right now, for both cAVS 1.8+ and ACE, the code will write a zero to the TDA register to signal completion on the receiving side of the connection.  After this patch is applied, it will set the value 0x8000000 to the same register on cAVS, and zero on ACE.

So... this seemingly has the effect of changing the cAVS behavior (which is believed to be working in-tree) and not ACE.  But the commit message says it's a fix for ACE.  Thus my confusion.